docs/source/basic/intro.rst

```diff
@@ -0,0 +1,90 @@
+
+   set(_TARGET_NAME helloWorld)
+   project(${_TARGET_NAME})
+
+   find_package(alpaka REQUIRED)
+
+   alpaka_add_executable(${_TARGET_NAME} helloWorld.cpp)
+   target_link_libraries(
+     ${_TARGET_NAME}
+     PUBLIC alpaka::alpaka)
+
+You can integrate alpaka into your project via ``find_package()`` in your ``CMakeLists.txt``.
+This requires, that you :doc:`install </basic/install>` alpaka.
+If you do not install alpaka in a default path such as ``/usr/local/`` you have to set the ``CMake`` argument ``-Dalpaka_ROOT=/path/to/alpaka/install``.
+
+The cmake configuration decides which ``alpaka::acc`` is available during compiling. For example, if you configure your ``cmake`` build with the CUDA backend (``cmake -DALPAKA_ACC_GPU_CUDA_ENABLE=ON``), ``cmake`` checks, if the CUDA SDK is available and if it found, the C++ template ``alpaka::acc::AccGpuCudaRt`` is available during compiling.
+
+What is alpaka
+--------------
+
+alpaka is ...
+~~~~~~~~~~~~~
+
+A Abstract Interface
+   It describing parallel execution on multiple hierarchy levels. It allows to implement a mapping to various hardware architectures but is no optimal mapping itself.
+
+Sustainably
+   *alpaka* decoupled the application from the availability of different accelerator frameworks in different versions, such as OpenMP, CUDA, HIP, etc. (50% on the way to reach full performance portability).
+
+Heterogeneity
+   An identical algorithm / kernel can be executed on heterogeneous parallel systems by selecting the target device. This allows the best performance for each algorithm and/or a good utilization of the system without major code changes.
+
+Maintainability
+   *alpaka* allows to provide a single version of the algorithm / kernel that can be used by all back-ends. There is no need for "copy and paste" kernels with different API calls for different accelerators. All the accelerator dependent implementation details are hidden within the *alpaka* library.
+
+Testability
+   Due to the easy back-end switch, no special hardware is required for testing the kernels. Even if the simulation itself will always use the *CUDA* back-end, the tests can completely run on a CPU. As long as the *alpaka* library is thoroughly tested for compatibility between the acceleration back-ends, the user simulation code is guaranteed to generate identical results (ignoring rounding errors / non-determinism) and is portable without any changes.
+
+Optimizable
+   Everything in *alpaka* can be replaced by user code to optimize for special use-cases.
+
+Extensible
+   Every concept described by the *alpaka* abstraction can be implemented by users. Therefore it is possible to non-intrusively define new devices, queues, buffer types or even whole accelerator back-ends.
+
+Data Structure Agnostic
+   The user can use and define arbitrary data structures.
+
+alpaka is not ...
+~~~~~~~~~~~~~~~~~
+
+An automatically optimal mapping of algorithms / kernels to various acceleration platforms
```

Comment:
Same point here, this one may be written as "Automatically optimizing mapping of ..." to match the style of the following items.

Reply:
Ànd this one I would write as "alpaka does not..." (instead of`is not`) and then the items as

- Automatically provide an optimal mapping of kernels to various acceleration platforms
- Automatically optimize concurrent data access
- Handle differences in arithmetic operations
- Guarantee determinism of results